pkg/fuzzy/googletranslate.go

```diff
@@ -0,0 +1,142 @@
+package fuzzy
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	"cloud.google.com/go/translate"
+	"github.com/spf13/viper"
+	"go.expect.digital/translate/pkg/model"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	"golang.org/x/text/language"
+	"google.golang.org/api/option"
+	htransport "google.golang.org/api/transport/http"
+)
+
+// --------------------Definitions--------------------
+
+// Interface that defines some of the methods of the Google Translate client.
+// This interface helps to mock the Google Translate client in unit tests.
+// https://pkg.go.dev/cloud.google.com/go/translate#Client
+type googleClient interface {
+	Translate(ctx context.Context, inputs []string, target language.Tag, opts *translate.Options) ([]translate.Translation, error) //nolint:lll
```

Comment:
alternatively, the types are self explanatory

```suggestion
	Translate(context.Context, []string, language.Tag, *translate.Options) ([]translate.Translation, error)
```